### DISCUSSION TITLE
If else

### DISCUSSION THREAD

**Discussion by Meenakshidhbw** [Meenakshidhbw/programming#7] [General]
This program checks the value of time and prints a message based on its range. If time is less than 10, it prints "Good morning."; if less than 18, it prints "Good day."; otherwise, it prints "Good evening."

**Comment by fjkun06** [comment]
I rememeber doing something like this back in the day. It's a great way to start building confidence in any programming language. Kudos.

**Comment by danii-07** [comment]
Hi Meena, that is soo cool!! Keep on coding girrl!! ;)

**Comment by Ngoc901** [comment]
Hi Meenakshi 👋

I will also try this one out! It's a great way to get more comfortable with programming! 🚀
